src/systems/ec/implementations/vanilla/packages/ec-preset-legacy-website/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@ecl/ec-preset-legacy-website",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.7.0",
+  "description": "ECL EC Legacy Website preset",
+  "main": "dist/scripts/ecl-ec-preset-legacy-website.js",
+  "style": "dist/styles/ecl-ec-preset-legacy-website.css",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build:copy": "ecl-builder copy",
+    "build:scripts": "ecl-builder scripts",
+    "build:styles": "ecl-builder styles",
+    "build": "rimraf build && run-p build:* -ln",
+    "clean": "rimraf build dist",
+    "dist": "npm run clean && cross-env NODE_ENV=production run-p build:* -ln",
+    "watch:scripts": "chokidar \"../../../../generic/**/*.js\" \"../../**/*.js\" -i \"/node_modules|build|dist|config\\.js/\" -c \"npm run build:scripts\"",
+    "watch:styles": "chokidar \"../../../../generic/**/*.scss\" \"../../**/*.scss\" -i \"/node_modules|build|dist/\" -c \"npm run build:styles\"",
+    "watch": "npm-run-all --parallel watch:* -ln",
+    "prepublish": "npm run dist"
+  },
+  "dependencies": {
+    "@ecl/ec-component-menu-legacy": "^2.7.0"
```

Comment:
should be a `devDependencies` (to be consistent with `@ecl/ec-preset-dev` and others)